tests/phpunit/payment-methods/class-wc-stripe-express-checkout-ajax-handler-test.php

```
@@ -98,4 +100,90 @@ public function provide_test_modify_country_locale_for_express_checkout() {
+		$output = ob_get_clean();
+
+		$response = json_decode( $output, true );
+
+		Ajax_Test_Helper::remove_hooks();
+
+		$this->assertIsArray( $response );
+		$this->assertArrayHasKey( 'success', $response );
+		$this->assertFalse( $response['success'] );
+		$this->assertArrayHasKey( 'data', $response );
+		$this->assertArrayHasKey( 'message', $response['data'] );
+	}
```

PHPMD (2.15.0)

[error] test_ajax_add_to_cart_returns_error_for_invalid_product accesses the super-global variable $_REQUEST. (undefined)

(Superglobals)